### PR TITLE
small tweaks (to support custom inspector)

### PR DESCRIPTION
**Pull request by mariakleiner**

### CODE REVIEW

File: pkg/ts/render/Composer.ts [hunk 1]
```diff
@@ -90,6 +91,8 @@ export class Composer extends EventEmitter {
   generateSlot(id, template, parent): Slot {
     return null;
   }
+  maybeReattachSlot(slot, container) {
```
Comment (mariakleiner) [inline pkg/ts/render/Composer.ts:94]: implementation in xen-composer:
```
  maybeReattachSlot(slot, container) {
    if (!slot.root.isConnected) {
      this.findContainer(container).appendChild(slot.root.host);
    }
  }
```

Comment (sjmiles) [reply pkg/ts/render/Composer.ts:94]: Ahhhh ... nice!

Comment (sjmiles) [reply pkg/ts/render/Composer.ts:94]: Probably we should be prepared for 'findContainer' to fail?

**Review by sjmiles — APPROVED** [review]

File: pkg/ts/render/Composer.ts (hunk 1 above)
Comment (mariakleiner) [reply pkg/ts/render/Composer.ts:94]: good point - made the change and created the corresponding PR:
https://team-review.git.corp.google.com/c/arcs-team/arcsjs-apps/+/1512785

**Pull request merged**